Resources/Prototypes/Catalog/Fills/Crates/bluespace_harvester.yml

```diff
@@ -0,0 +1,347 @@
+      amount: 3
+
+- type: entity
+  id: CrateHarvesterBackpackHolding
+  parent: CrateIndustrialSecure
+  suffix: Bluespace Harvester, Backpack Holding
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomHarvesterBackpackHolding
+      amount: 2
+
+- type: entity
+  id: CrateHarvesterAmeJar
+  parent: CrateIndustrialSecure
+  suffix: Bluespace Harvester, AmeJar
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: AmeJar
+      amount: 10
+
+- type: entity
+  id: CrateHarvesterCircuitboard
+  parent: CrateIndustrialSecure
+  suffix: Bluespace Harvester, Circuitboard
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomHarvesterCircuitboard
+      amount: 3
+
+# Technological Loot
+- type: entity
+  id: CrateHarvesterResearchDisk
+  parent: CrateTechnologicalSecure
+  suffix: Bluespace Harvester, ResearchDisk
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomHarvesterResearchDisk
+      amount: 4
+
+- type: entity
+  id: CrateHarvesterTechnologyDisk
+  parent: CrateTechnologicalSecure
+  suffix: Bluespace Harvester, TechnologyDisk
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomHarvesterTechnologyDisk
+      amount: 7
+
+- type: entity
+  id: CrateHarvesterAnomaly
+  parent: CrateTechnologicalSecure
+  suffix: Bluespace Harvester, Anomaly
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomAnomalySpawner
+      amount: 1
+
+- type: entity
+  id: CrateHarvesterArtifact
+  parent: CrateTechnologicalSecure
+  suffix: Bluespace Harvester, Artifact
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomArtifactSpawner
+      amount: 1
+    
+- type: entity
+  id: CrateHarvesterAnomalyKit
+  parent: CrateTechnologicalSecure
+  suffix: Bluespace Harvester, AnomalyKit
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomHarvesterAnomalyKit
+      amount: 4
+
+- type: entity
+  id: CrateHarvesterEncryptionKey
+  parent: CrateTechnologicalSecure
+  suffix: Bluespace Harvester, EncryptionKey
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomHarvesterEncryptionKey
+      amount: 5
+
+- type: entity
+  id: CrateHarvesterBorg
+  parent: CrateTechnologicalSecure
+  suffix: Bluespace Harvester, Borg
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomHarvesterBorg
+      amount: 3
+
+- type: entity
+  id: CrateHarvesterMech
+  parent: CrateTechnologicalSecure
+  suffix: Bluespace Harvester, Mech
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: RandomHarvesterMech
+      amount: 2
+
+- type: entity
+  id: CrateHarvesterReagentSlime
+  parent: CrateLimitedSecure
+  suffix: Bluespace Harvester, ReagentSlime
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: ReagentSlimeSpawner
+      amount: 20
```

Comment:
It's fakking terrible, omg
the anomaly creates about 7 pieces when it explodes, and that's a huge danger to the station, and there are 20 of them!

Reply:
There may be a singularity in one of the dangerous boxes, I think 20 slimes still won't match it, but it's still dangerous.